### ISSUE TITLE
(short issue description)

### ISSUE THREAD

**Issue by leongross** [aws/aws-cli#7351]
### Describe the bug

Querying for certain attributes of a virtual machine results in an error: `An error occurred (InvalidRequest) when calling the DescribeImageAttribute operation: The request received was invalid.`

Querying for the all attributes of the same image succeeds. The attribute can be found in the query output. 

I observed this behavior for the publicly available, official Windows Image with AMI-ID `ami-09e7c7f5617a47830`. I searched for the `TpmSupport` attribute.

### Expected Behavior

Receiving the output:
```
{
    "TpmSupport": "v2.0"
}
```

### Current Behavior

`An error occurred (InvalidRequest) when calling the DescribeImageAttribute operation: The request received was invalid.`

## Getting all attributes
```
aws ec2 describe-images --region us-east-2 --image-id ami-09e7c7f5617a47830 
```

Searching for `TpmSupport` yields: `"TpmSupport": "v2.0",`

## Getting the `TmpSupport` attribute
```
aws ec2 describe-image-attribute --region us-east-2 --image-id ami-09e7c7f5617a47830 --attribute TpmSupport
An error occurred (InvalidRequest) when calling the DescribeImageAttribute operation: The request received was invalid.
``` 

### CLI version used

aws-cli/2.7.34 Python/3.9.11 Linux/6.0.2-arch1-1 exe/x86_64.arch prompt/off

### Environment details (OS name and version, etc.)

OS: `Arch Linux`, kernel version `6.0.2-arch-1`

**Comment by tim-finnigan** [comment]
Hi @leongross - in the CLI documentation for this command (https://awscli.amazonaws.com/v2/documentation/api/latest/reference/ec2/describe-image-attribute.html) it notes that the possible values include `tpmSupport` rather than `TpmSupport`. If you use `tpmSupport` then the command should work as intended. Please let us know if you run into any issues after trying that.

**Comment by leongross** [comment]
I tried to execute the command as you said:
```sh
aws ec2 describe-image-attribute --region us-east-2 --image-id ami-09e7c7f5617a47830 --attribute tpmSupport
```

This results in the following error:
```
An error occurred (AuthFailure) when calling the DescribeImageAttribute operation: Not authorized for image:ami-09e7c7f5617a47830
```

Further I again used the command `describe-images`, which again yields the information.
So also with the changed capitalization an error occurs.

**Comment by tim-finnigan** [comment]
> So also with the changed capitalization an error occurs.

Yes this new error involves Authorization and please note that this is a server-side error rather than client side issue with the AWS CLI directly. So in cases like these we recommend reaching out to [AWS Support](https://aws.amazon.com/contact-us/) and/or asking in the [re:Post forums](https://repost.aws/) if you can't find an existing answer to your issue. (That way your question is more likely to get directed to someone with more domain knowledge on the specific service you're using - in this case EC2).

I can also forward questions internally to the EC2 team though that process usually takes a little more time. If you'd like me to do that could you please provide more information about the AMI you are using and the permissions of the profile you are using?
